src/components/accessories/patientTherapy/therapyForm/TherapyForm.tsx

```diff
@@ -191,7 +200,7 @@ const TherapyForm: FC<TherapyProps> = ({
               <DateField
                 fieldName="startDate"
                 fieldValue={formik.values.startDate}
-                disableFuture={true}
+                disableFuture={false}
```

Comment:
Is this assertion right? Don't we need the future date?

Reply:
Here I use the false value to display future date. When it is true the future dates don't display.